channels/rail/rail_common.h

```diff
@@ -28,22 +28,33 @@
 
 extern const char* const RAIL_ORDER_TYPE_STRINGS[];
 
-#define RAIL_PDU_HEADER_LENGTH			4
+#define RAIL_PDU_HEADER_LENGTH	4
 
 /* Fixed length of PDUs, excluding variable lengths */
-#define RAIL_HANDSHAKE_ORDER_LENGTH		4	/* fixed */
-#define RAIL_HANDSHAKE_EX_ORDER_LENGTH		8	/* fixed */
-#define RAIL_CLIENT_STATUS_ORDER_LENGTH		4	/* fixed */
-#define RAIL_EXEC_ORDER_LENGTH			8	/* variable */
-#define RAIL_SYSPARAM_ORDER_LENGTH		4	/* variable */
-#define RAIL_ACTIVATE_ORDER_LENGTH		5	/* fixed */
-#define RAIL_SYSMENU_ORDER_LENGTH		8	/* fixed */
-#define RAIL_SYSCOMMAND_ORDER_LENGTH		6	/* fixed */
-#define RAIL_NOTIFY_EVENT_ORDER_LENGTH		12	/* fixed */
-#define RAIL_WINDOW_MOVE_ORDER_LENGTH		12	/* fixed */
-#define RAIL_GET_APPID_REQ_ORDER_LENGTH		4	/* fixed */
-#define RAIL_LANGBAR_INFO_ORDER_LENGTH		4	/* fixed */
-#define RAIL_LANGUAGEIME_INFO_ORDER_LENGTH		44	/* fixed */
+#define RAIL_HANDSHAKE_ORDER_LENGTH				4		/* fixed */
```

Comment:
why reformat ?

Reply:
I didn't just reformat, I added the rest of the order's length and had to reformat because the longer names that were added